tests/support/__init__.py

```diff
@@ -158,6 +158,11 @@ def assertFileContentIdentical(self, file_actual, file_expected):
                     os.path.relpath(file_actual, MIG_BASE),
                     ''.join(different_lines)))
 
+    def assertFileExists(self, relative_path):
+        path_kind = self.assertPathExists(relative_path)
+        assert path_kind == "file", "expected a file but found %d" % (path_kind,)
```

Comment:
I'm quite sure this should be `%s` rather than `%d` so I'll fix it in merge and add the missing docstring while at it.